server/api/api/endpoints/nuclio.py

```diff
@@ -587,6 +634,25 @@ def _handle_nuclio_deploy_status(
     )
 
 
+async def _get_api_gateways_hosts_for_function(
```

Comment:
I think the meaning is clearer with:
```suggestion
async def _get_api_gateways_urls_for_function(
```